dispatcher/main.go

```diff
@@ -105,7 +105,8 @@ func main() {
 		}
 
 		owners := string(secret.Data["owners"])
-		reportutils.Report(run, strings.Split(owners, ","))
+		templateURL := string(secret.Data["email.path.template"])
```

Comment:
We need a fallback. If there isn't a path in the secret, log an error and exit gracefully.